Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "grcov"
-version = "0.8.18"
```

Comment:
Could you revert this change? I'll bump the version in a later patch

Reply:
done!